File: app/src/main/java/com/casebeaumonde/UpdateProfilePicResponse.java
package com.casebeaumonde;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class UpdateProfilePicResponse {

@SerializedName("code")
@Expose
private Integer code;
@SerializedName("data")
@Expose
private Data data;

public Integer getCode() {
return code;
}

public void setCode(Integer code) {
this.code = code;
}

public Data getData() {
return data;
}

public void setData(Data data) {
this.data = data;
}

    public class Data {

        @SerializedName("image")
        @Expose
        private String image;

        public String getImage() {
            return image;
        }

        public void setImage(String image) {
            this.image = image;
        }

    }
}

File: app/src/main/java/com/casebeaumonde/UpdateProfilePicResponse.kt
package com.casebeaumonde

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class UpdateProfilePicResponse {

    @SerializedName("code")
    @Expose
    private var code: Int? = null

    @SerializedName("data")
    @Expose
    private var data: Data? = null

    fun getCode(): Int? {
        return code
    }

    fun setCode(code: Int?) {
        this.code = code
    }

    fun getData(): Data? {
        return data
    }

    fun setData(data: Data?) {
        this.data = data
    }

    class Data {
        @SerializedName("image")
        @Expose
        var image: String? = null
    }
}